.claude/skills/konflux-onboarding/SKILL.md

```
@@ -354,28 +355,141 @@ For Type A packages, check if `manifests/modular-architecture/modules/<name>/` e
+git clone --depth 1 <fork-url> "$WORK_DIR/openshift-release"
+cd "$WORK_DIR/openshift-release"
+git remote add upstream https://github.com/openshift/release.git
+git fetch upstream main
+git checkout -b <component-name>-openshift-ci upstream/main
 ```
 
+### Step 6b: Modify the Configuration File
+
+Edit **only** the following file:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)